libraries/GCS_MAVLink/GCS_Param.cpp

```diff
@@ -47,11 +47,10 @@ GCS_MAVLINK::queued_param_send()
     const uint32_t tnow = AP_HAL::millis();
     const uint32_t tstart = AP_HAL::micros();
 
-    // use at most 30% of bandwidth on parameters. The constant 26 is
-    // 1/(1000 * 1/8 * 0.001 * 0.3)
```

Comment:
Why did we lose this crafted constant rather than doing another*1024?

I did work out what this derivation meant at one stage.  I should have written that down / adjusted this comment.

Reply:
since it appears that this calculus was for bits and not bytes

just redo the calculus with the current units